apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -70,28 +63,23 @@ class ItineraryCard extends React.Component<Props, State> {
       <Card
         style={[
           styles.container,
-          (hovered || detailOpened) && styles.containerElevated,
+          (isHovered || detailOpened) && styles.containerElevated,
```

Comment:
you need to add `onMouseEnter` and `onMouseLeave` to this component for it to work.